Source/web_server/endpoints/studio.py

```
@@ -1,7 +1,11 @@
-from web_server._logic import web_server_handler, server_path
-import util.versions as versions
-import time
+# Standard library imports
 import json
+import time
+
+# Local application imports
+import util.versions as versions
```

Ruff (0.8.2)

`util.versions` imported but unused

Remove unused import: `util.versions`

(F401)